parser-typechecker/src/Unison/Util/ColorText.hs

```diff
@@ -95,6 +96,7 @@ toANSI (AnnotatedText chunks) =
     HiCyan   -> [ANSI.SetColor ANSI.Foreground ANSI.Vivid ANSI.Cyan]
     HiWhite  -> [ANSI.SetColor ANSI.Foreground ANSI.Vivid ANSI.White]
     Bold     -> [ANSI.SetConsoleIntensity ANSI.BoldIntensity]
+    Underline -> [ANSI.SetUnderlining ANSI.SingleUnderline]
```

Comment:
I didn't remember ANSI underlining was a thing.  Can you add a screen shot?

Reply:
It looks like an underline. :) Here's a screenshot https://i.stack.imgur.com/Kw4Yf.png

Reply:
Lol, thanks, but I was interested in a screenshot of the ucm output that uses it — currently the only output samples are the transcript which doesn’t include the ansi sequences.